scripts/soroban_repo_to_horizon_repo.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# soroban_repo_to_horizon_repo.sh - given a soroban branch to horizon branch
+#
+# Syntax:   soroban_repo_to_horizon_repo.sh <soroban_branch_name>
+#
+# Examples: soroban_repo_to_horizon_repo.sh main
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "Syntax: soroban_repo_to_horizon_repo.sh <soroban_branch_name>"
```

Comment:
another nitpick: you can use `$0` here instead of hard-coding the script name

Reply:
I like this suggestion.